### PR TITLE
Add Hindi support - v3.8.1

### PR DESCRIPTION
**Pull request by BitESKrunker**
- Added Hindi (हिन्दी) translation file.
- Added contributors.

### CODE REVIEW

**Comment by mty22** [comment]
Thanks! I've asked for some native speakers to review.

**Comment by mty22** [comment]
pending review from native speakers

**Comment by BitESKrunker** [comment]
I can get Jytesh, a KDH Discord server mod to review it if you like!

**Comment by BitESKrunker** [comment]
Hey Matt, 
Any updates? If not, ill look for a reviewer whos fluent in Hindi

**Comment by mty22** [comment]
@BitESKrunker sure, if you can send them to me; that'd work out well. frzn#1030

**Comment by mty22** [comment]
https://krunker.io/social.html?p=profile&q=zeude /  ZEUDE#3733 reached out to me via @BitESKrunker -- he'll review this.

**Comment by zeude** [comment]
Alright, I will be reviewing the Hindi support and will hit you up with the changes asap.

**Comment by BitESKrunker** [comment]
[Hindi_Support_Krunker.docx](https://github.com/mty22/krunker.io-translations/files/6294456/Hindi_Support_Krunker.docx)

Have gotten the reviews from @zeude, the team is now going over the original hi.js and fixing all the mistakes. Great work by Zeude, made loads of improvements as well as making it easier for lower ages to understand the game easier.

**Comment by BitESKrunker** [comment]
Hey @mty22!
Hi.js is now updated [https://github.com/mty22/krunker.io-translations/pull/17/commits/3cdaf401a038d1813929506f7df2dc166e295e0f] All the mistakes and simple additions have been added.

File: README.md
Comment (mty22) [inline README.md:38]: @BitESKrunker  Can you please add ZEUDE here as a reviewer?

**Comment by zeude** [comment]
I have reviewed the project and everything looks fine. Thank you @BitESKrunker and @mty22 for giving me this opportunity to contribute to the game. Hoping to contribute more in the future.

File: hi.js
Comment (mty22) [inline hi.js:64]: Let's revert this one back to English.

The grammar will appear as:

`You're {1} मार लकीर पर`

or

`Player is एक {1} मार लकीर पर`

Comment (mty22) [reply hi.js:64]: https://github.com/mty22/krunker.io-translations/blob/main/en.js#L64

File: hi.js
Comment (mty22) [inline hi.js:87]: Is this not translatable ? Or best to be kept in English?

**Review by mty22 — CHANGES_REQUESTED** [review]
Just a few minor changes.

Also @BitESKrunker - can you please provide me with an accurate work breakdown in percentages, including the work that @zeude put in?

**Example**:

BitES - 30% (translation)
Milerobloxfan - 30%  (translation)
Shrekajerk - 15% (review)
ZEUDE - 25% (review/translation fixing)

File: hi.js
Comment (zeude) [reply hi.js:87]: I would suggest keeping this in English only, cuz there is no intelligible alternate in Hindi.

File: hi.js
Comment (HistoriFy) [reply hi.js:64]: The actual meaning of the word "लकीर" is "a straight drawn line". Another Google Translation Error. 

Better to keep it as kill streak only

Comment (BitESKrunker) [reply hi.js:64]: So we keep it as killstreak only? Or should I change it to what Matt [ `You're {1} मार लकीर पर` ] has asked to revert to?

**Review by BitESKrunker — COMMENTED** [review]
> Just a few minor changes.
> 
> Also @BitESKrunker - can you please provide me with an accurate work breakdown in percentages, including the work that @zeude put in?
> 
> **Example**:
> 
> BitES - 30% (translation)
> Milerobloxfan - 30% (translation)
> Shrekajerk - 15% (review)
> ZEUDE - 25% (review/translation fixing)


```
BitES (Translator) - 33%
Shrekajerk (Translator) - 33%
Milerobloxfan (Translator) - 33%
Zeude (Reviewer) - 1%
```

File: hi.js
Comment (BitESKrunker) [reply hi.js:87]: I tried finding alternatives but no luck, so I decided to keep it in English.

File: hi.js
Comment (mty22) [reply hi.js:64]: Just revert it to English as it needs to be accurate.

Google Translate, whilst being a fantastic tool, really should not be used in projects like this. It does not have true understanding of each word as they're not full sentences, thus no context to work from without having knowledge of the game.

**Comment by BitESKrunker** [comment]
> Just a few minor changes.
> 
> Also @BitESKrunker - can you please provide me with an accurate work breakdown in percentages, including the work that @zeude put in?
> 
> **Example**:
> 
> BitES - 30% (translation)
> Milerobloxfan - 30% (translation)
> Shrekajerk - 15% (review)
> ZEUDE - 25% (review/translation fixing)

```
BitES - 25% (Translation)
Milerobloxfan - 25% (Translation)
Shrekajerk - 25% (Translation)
Zeude - 25% (Translation)
```

**Comment by BitESKrunker** [comment]
> Just a few minor changes.
> 
> Also @BitESKrunker - can you please provide me with an accurate work breakdown in percentages, including the work that @zeude put in?
> 
> **Example**:
> 
> BitES - 30% (translation)
> Milerobloxfan - 30% (translation)
> Shrekajerk - 15% (review)
> ZEUDE - 25% (review/translation fixing)

```
BitES - 25% (Translation)
Milerobloxfan - 25% (Translation)
Shrekajerk - 25% (Translation)
Zeude - 25% (Translation)
```

**Comment by mty22** [comment]
I've received feedback in DM from various players in regards to the translation quality of this PR. I need confidence that everything is accurate before it will be merged.

In addition, the forked branch needs to be rebased before merge as upstream has changed.

File: hi.js
Comment (HooD-ii) [inline hi.js:20]: Change this dude

**Comment by mty22** [comment]
Just a note - we've had more community members willing to assist with this translation.

Given that the nature of this is open source, you're all welcome to contribute where possible. To avoid duplicating work, please wait for this pull request to be merged (Krunker patch v3.8.1).

Alternatively, reach out to the current team of contributors for Hindi language advising you're available to assist:

https://github.com/mty22/krunker.io-translations/pull/17#issuecomment-818437487

+ historify.

**Comment by improplayer** [comment]
@BitESKrunker @mty22 @zeude  Can i Help out with the translations or is it too late? IM'SLASHY#0001 (nitro might expire still ill be IM'SLASHY

**Comment by mty22** [comment]
This is delayed due to COVID in India.

No rush guys, supporting your family and friends is incredibly important.

**Comment by VandyTweaks** [comment]
I just reviewed the hindi.js file and I think we can use hindi words instead of writing them in english (using devnagri)

**Comment by HistoriFy** [comment]
> I just reviewed the hindi.js file and I think we can use hindi words instead of writing them in english (using devnagri)
> 
> I can help

Refer to this Reply Once Please = https://github.com/mty22/krunker.io-translations/pull/17#issuecomment-826059532

the Entire translation will be updated by today.

**Comment by BitESKrunker** [comment]
The final release of hi.js was released, waiting to add all contributors list in percentages.

**Comment by mty22** [comment]
```
zeude = 20 % https://krunker.io/social.html?p=profile&q=zeude
Histo = 20 % https://krunker.io/social.html?p=profile&q=HistoriFy
Keval = 15 % https://krunker.io/social.html?p=profile&q=TheDevKeval
Slog = 15 % https://krunker.io/social.html?p=profile&q=slogllykop
Bites = 5 % https://krunker.io/social.html?p=profile&q=BitES
Mile = 10 % https://krunker.io/social.html?p=profile&q=milethebot
Shrek = 15 % https://krunker.io/social.html?p=profile&q=le_hem
```

**Comment by mty22** [comment]
I need to fix a few things (ie. trim whitespace on end of lines), then good to merge.

**Comment by mty22** [comment]
```@ZEUDE#3733 @Histo#1005 @Keval#8167 @Slog#8870 @BitES#7783 @bentain10officalfan#4417 @shrekajerk#2387```

**Comment by mty22** [comment]
KB roles sorted.

**Pull request merged**
